### ISSUE TITLE
Swagger response validation and still relative $refs doesn't work properly

### ISSUE THREAD

**Issue by k911** [FieryCod/KNIT-homepage-back#16]
* Referencing via json-ref from model/Article.yaml to the object in the same file.
`$ref: '#/ArticleContent'`
`$ref: 'model/Article.yaml#/ArticleContent'`
doesn't work.

Error thrown (for `$ref: 'model/Article.yaml#/ArticleContent'` was similar)
```
API Errors:
 ✘
 ✘     #/paths/~1articles/get/responses/200/schema/items/properties/content/$ref: Articleconten could not be resolved: #/ArticleContent
 ✘     #/paths/~1articles~1{id}/get/responses/200/schema/properties/content/$ref: Articleconten could not be resolved: #/ArticleContent
 ✘
 ✘   2 errors and 0 warnings
 ✘
```
But referencing fille by absolute path like:
`
$ref: 'http://localhost:3000/api/swagger/models/Article.yaml#/ArticleContent'
`
Does work.

- ~~ Also swagger validation of responses don't work on objects.~~

- [x] Response validation
- [x] Relative $refs outside swagger.yaml

**Comment by k911** [comment]
Fixed first one in: https://github.com/FieryCod/KNIT-homepage-back/commit/3387a16b5b6018ea2b7ddecd6ee7860bfa711015

**Comment by FieryCod** [comment]
Ok I figured out that when we point with $ref to the definition in the same file(except from swagger.yaml) $ref cannot find definition.

For example:
We got an Article.yaml and we're defining schema of Article in that file. Supposing we got properties like: content, createdAt etc.. Let the content be an object which we wanna define in the same file and $ref to it.
Such a $ref does not work. 

**Proposed solution:** If we define ArticleContent in the another file and then in Article we will point to that file with $ref I does not outputs and error.
